src/tabs/ConstantsPage.js

```diff
@@ -9,53 +9,180 @@
-          title="Configurar bluetooth"
-        />
-      </CellContainer>
-
-      <CellContainer>
-        <SecondaryButton
-          onPress={() => GoToTab('HistoryPage')}
-          title="Página de Históricos"
-        />
-      </CellContainer>
-
-      <CellContainer>
-        <SecondaryButton
-          onPress={() => GoToTab('ConstantsPage')}
-          title="Página de Constantes"
-        />
-      </CellContainer>
-
-      <CellContainer>
-        <SecondaryButton
-          onPress={() => GoToTab('EvaluationPage')}
-          title="Página de testes"
-        />
-      </CellContainer>
-
-      <CellContainer>
-        <SecondaryButton
-          onPress={() => GoToTab('ControlPage')}
-          title="Página para controlar o Tracer"
-        />
-      </CellContainer>
+    <SafeAreaView style={styles.container}>
+
+      <View backgroundColor={COLORS.gray4}>
+        <View style={ROTATION.OitoEMeia}>
+          <View style={styles.titleContainer} paddingLeft = {20}>
+            <H3 color="white">SETUP</H3>
+          </View>
+        </View>
+      </View>
+
+      {/* Dinamic constants table*/}
+      <View style={styles.constantsInputContainer}>
+        <ScrollView>
+          {constantCouples.map((element, index) => {
+            return (
+              <View style = {styles.tableRow} key={index}>
+
+                <View style = {styles.tableCell}>  
+                  <View style = {styles.textView}>
+                    <Body>{element[0]}</Body>
+                  </View>
+
+                  <View style = {styles.textInputView}>
+                    <TextInput style={styles.textInput}/>
+                  </View>
+                </View>
+                
+                <View style = {styles.tableCell}>  
+                  <View style = {styles.textView}>
+                    <Body>{element[1]}</Body>
+                  </View>
+                  
+                  <View style = {styles.textInputView}>
+                    <TextInput style={styles.textInput}/>
+                  </View>
+                </View>
+
+                <View style = {styles.tableCell}>  
+                  <View style = {styles.textView}>
+                    <Body>{element[2]}</Body>
+                  </View>
+                  
+                  <View style = {styles.textInputView}>
+                    <TextInput style={styles.textInput}/>
+                  </View>
+                </View>
+
+              </View>
+            );
+          })}
+        </ScrollView>
+      </View>
+
+      <View style={styles.buttonsContainer}>
+        <View style={styles.buttonRow}>
+          <View style = {styles.buttonCell}>  
+            <ActionButton title="Enviar" />
+          </View> 
+          <View style = {styles.buttonCell}>  
+            <GreenActionButton title="Salvar" />
+          </View> 
+          <View style = {styles.buttonCell}>  
+            <RedActionButton title="Clear" />
```

Comment:
Eu particularmente acho estranho deixar os demais botões em português e esse em inglês. Isso é realmente bem batata, mas por uma questão de padronização, acharia melhor ao invés de "Clear", deixar algo como "Limpar". Comentário bem inútil mesmo, perdão kkkkkkkkk

Reply:
Nice, acho que isso leva a uma boa discussão que ainda não tivemos, se o Monitracer será em PT ou EN. Por hora vou colocar os botões superiores em PT, mas os de baixo não podem receber palavras mto grandes sem ficar com design feio Tururu.